README.md

```diff
@@ -3,13 +3,13 @@
 ## Definitions
 Define the following: 
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion: a method that calls itself, it is the opposite of iterrative
```

Comment:
I don't think I'd say "the opposite of iterative" though maybe that's true? Either way, I wouldn't make that statement. I might say "opposed to iterative"